src/software/ai/evaluation/ball.cpp

```diff
@@ -25,3 +25,16 @@ bool ballInEnemyCorner(const Field &field, const Ball &ball, double radius)
             (distance(ball.position(), field.enemyCornerNeg()) < radius)) &&
            field.pointInFieldLines(ball.position());
 }
+
+bool hasBallBeenKicked(const Ball &ball, const Angle &expected_kick_direction,
+                       double min_kick_speed)
+{
+    static constexpr double MAX_ANGLE_DIFFERENCE = 20;
```

Comment:
nit: add units, or store this constant as an `Angle`

Reply:
changed to store constant as `Angle`